### ISSUE TITLE
WebView2 crash when CTRL+F search is open and webview is disposed.

### ISSUE THREAD

**Issue by frankdekker** [MicrosoftEdge/WebView2Feedback#2807]
**Description**
WebView will throw exception and crash when CTRL+F dialog is open, and the view is disposed.

Extracted a part of our application as separate project: https://github.com/frankdekker/WebView2MoveFocusCrash

**Version**
SDK: 1.0.1340-prerelease
Runtime: Edge 105.0.1343.33
Framework: WPF
OS: Win10

**Repro Steps**
See readme of the example repo. 

1) Build the application and then start WebViewExample
2) In the first tab where wikipedia is shown, press CTRL+F to show the search dialog.
3) Now click the close button of that tab to close that tab, this will close tab and activate the next tab.
![image](https://user-images.githubusercontent.com/2179983/190856773-1d6d689f-8dbd-4f7e-a825-7fa2291d484c.png)

4) Exception will be thrown
```
System.InvalidOperationException: CoreWebView2 members cannot be accessed after the WebView2 control is disposed.
 ---> System.Runtime.InteropServices.COMException (0x8007139F): The group or resource is not in the correct state to perform the requested operation. (0x8007139F)
   at Microsoft.Web.WebView2.Core.Raw.ICoreWebView2Controller.MoveFocus(COREWEBVIEW2_MOVE_FOCUS_REASON reason)
   at Microsoft.Web.WebView2.Core.CoreWebView2Controller.MoveFocus(CoreWebView2MoveFocusReason reason)
   --- End of inner exception stack trace ---
   at Microsoft.Web.WebView2.Core.CoreWebView2Controller.MoveFocus(CoreWebView2MoveFocusReason reason)
   at Microsoft.Web.WebView2.Wpf.WebView2.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndHost.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
```

**Comment by incheon-kim** [comment]
This happens too on 1.0.1343.22, WPF, Win 10.

**Comment by xu-ms** [comment]
@incheon-kim @frankdekker 
Some similar issues are fixed in runtimes 107.0.1418.11+ and 108.0.1432.0+.
I cannot reproduce the issue in runtime 108.0.1462.54. 
Could you update runtime and check the issue can still be reproduced?

**Comment by victorhuangwq** [comment]
Looks like the issue has been addressed - I will close the issue for now. Feel free to open a new issue if necessary
